README.md

```diff
@@ -0,0 +1,49 @@
+# animate
+Anisotropic mesh adaptation toolkit for Firedrake
+
+
+    The implementation of metric-based mesh adaptation used in PETSc assumes that the
+    metric is piece-wise linear and continuous, with its degrees of freedom at the
+    mesh vertices.
```

Comment:
Perhaps put this paragraph after the next one since it is more about details? Also, don't use code styling.

Reply:
I borrowed the description from the previous meshadapt.py implementation in Firedrake for continuity. You are right, I incorporated it into the overall description and forgot to delete the original copy.